node/node-nagios.inc.php

```diff
@@ -148,7 +159,11 @@ function detail($node, $short)
                     echo '<td>'.$dt_user.'</td>';
                     echo '<td class="fixed">'.$dt_reason."</td>";
                     echo '<td>';
-                    echo '<a class="delete" href="'.sprintf($NAGIOS_URL, $server).'/cgi-bin/cmd.cgi?cmd_typ=78&amp;down_id='.$dt_id.'" />';
+                    if (strpos($server, 'icinga') !== -1) {
+                        echo '&hellip;';
```

Comment:
There should be a way to delete downtimes from Icinga - looks like this is not a simple URL but rather a form. Don't know whether it's worth looking into this more?

Reply:
IIRC it's more an issue that the previous method worked well due to the use of X509 auth, whereas more often than not Icinga2 dumps you on a login page and the context is lost.

Reply:
Ah I see, in which case I'll resolve this.